### ISSUE TITLE
Shiny con los primeros datos que estén en el SiAgro

### ISSUE THREAD

**Issue by AliciaMstt** [CONABIO/GEF-Agrobiodiversity#16]
En la reunión 17 julio 2020 con Vicente, Fran, et al, acordamos que se haría una Shinyapp que demostrara el potencial del SiAgro y la utilidad de Zendro como parte de este.

Comenzar por una shiny de los primeros datos que estén en SiAgro (grupos focales, nutri, o  colectas, maíces yolanda, pensar en datos interconectados).

**Comment by APonce73** [comment]
Lo más parsimonioso podría ser GF que se tiene la lista de especies y lo de nutrición. Se podría ver que tan balanceada es la dieta en cada localidad.

**Comment by AliciaMstt** [comment]
Estoy de acuerdo. No urge, pero fui haciendo el issue para que no se nos olvidara en el futuro mediano. Gracias :)
